Makefile

```diff
@@ -2,16 +2,16 @@
 .PHONY: build clean sdist test
 
 build:
-	pip3 install -r requirements.txt
+	pipenv install
```

Comment:
I wouldn't normally expect this to be in the Makefile. It's an environment setup step and I would have expected it to be in the readme.

Reply:
I've left it here, but updated the readme on how what the alternative ways to build the environment